joomla/joomla-cms
# Remove hard-limit of 20 to the Module Tags

Pull Request for Issue #36016.

### Summary of Changes

Removed the hard limit for Module Tags-Popular and Module Tags-Similar.


### Testing Instructions
1. Site modules--> New-->Tags-Popular.
2. Set the maximum tags to a number > 20 and make sure that the tags are displayed correctly in the frontend.


### Actual result BEFORE applying this Pull Request
see #36016


### Expected result AFTER applying this Pull Request
Any number is possible.



### Documentation Changes Required
